themes/edx.org/lms/templates/dashboard.html

```diff
@@ -180,6 +180,19 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
       </ul>
     </section>
   </section>
+  % if xseries_credentials:
```

Comment:
Question for you guys (cc: @jimabramson) -- I took a look at the source of the 'xseries_credentials' parameter and noticed that it derives from a private helper method located in common/djangoapps/student/views.py (_get_xseries_credentials.py).  It seems odd to me that this method exists, because what we've essentially done is introduced an installation-specific concept into our general-purpose platform.  That is, no other Open edX operator is going to want this workflow to execute in their installation, because they do not (and should not) have 'xseries' program categories in their setup.  After talking with @douglashall and @mjfrey we think it makes sense to move this into configuration -- ideally via ConfigurationModel but also acceptably via settings file.

I don't consider it a blocker for this particular PR, but it would be great to see a ticket logged and the code refactored to stick with the generic nature of the platform.